cl/recap/tests.py

```diff
@@ -890,23 +869,38 @@ def setUp(self) -> None:
         )
         self.docs = [self.doc12, self.doc12_att1]
 
+    @classmethod
+    def setUpTestData(cls) -> None:
+        filename = "1-20-cv-10189-FDS.zip"
+        user = User.objects.get(username="recap")
+
+        cls.pq = ProcessingQueueFactory.create(
+            id=11,
```

Comment:
Another ID. They're sneaky, eh?

Also, I think by convention this method usually comes before the setUp method.

Reply:
√